cmd/user-api/main.go

```diff
@@ -2,8 +2,9 @@ package main
 
 import (
 	"github.com/gin-gonic/gin"
-	"github.com/sokangho-wex/ps-tag-onboarding-go/internal/handlers"
-	"github.com/sokangho-wex/ps-tag-onboarding-go/internal/handlers/validators"
+	errorhandler "github.com/sokangho-wex/ps-tag-onboarding-go/internal/domain/onboardingerrors/httphandler"
+	userhandler "github.com/sokangho-wex/ps-tag-onboarding-go/internal/domain/users/httphandler"
```

Comment:
Ah! this is something I didn't catch first time around. we should try to avoid aliasing packages as much as possible.  this kind of ruins my subpackaging idea.  Commented below on ways to improve.  Sorry we might have to undo one of my ideas from yesterday.

Reply:
Thanks, fixed in the new commit